IdfaIOS/CustomGps.m

```diff
@@ -0,0 +1,72 @@
+#import <CoreLocation/CoreLocation.h>
+
+//Please ensure this path points to the file 'Assets/Plugins/iOS/Amplitude/Amplitude.h'
+#import "../../Plugins/iOS/Amplitude/Amplitude.h"
+
+#import "../../Plugins/iOS/Amplitude/AmplitudeCWrapper.h"
+
+typedef NSDictionary* (^AMPLocationInfoBlock)(void);
+
+@interface CustomLocationManager : NSObject<CLLocationManagerDelegate> 
+@property (nonatomic, strong) CLLocationManager *locationManager;
+@property (nonatomic, strong) NSString *instanceName;
+@end
+
+@implementation CustomLocationManager 
+
+- (instancetype)initWithString:(NSString *)instanceNameString {
```

Comment:
change the name to `init:(NSString *)instanceName`.
Usually you can skip `with` if there's just one param.